docs/experiments-microblog.md

```diff
@@ -13,6 +13,7 @@ nohup sh target/appassembler/bin/IndexCollection -collection TwitterCollection
 /path/to/Tweets2011/ -generator JsoupGenerator  -index lucene-index.Tweets2011.pos+docvectors -threads 32 \
 -storePositions -storeDocvectors -optimize > log.Tweets2011.txt &
 ```
+__NB:__ The process is backgrounded 
```

Comment:
I know this seems like I'm hammering on a (rather non-crucial) point, but if it's backgrounded how do I tell it's finished? Will `log.Tweets2011.txt` have a line at the end saying it's done? Or do I have to check with ps/top/htop?